File: src/main/scala/qt/quickcontrols2/QQuickStyle.scala
package qt.quickcontrols2

import scalanative.unsafe.*
import qt.QString
import scala.util.Using.Manager

@extern
@link("Qt5QuickControls2")
object _QQuickStyle:
  def q_quick_style_set_style(style: QString): Unit = extern

object QQuickStyle:
  def setStyle(style: String)(using m: Manager): Unit = 
    _QQuickStyle.q_quick_style_set_style(QString(style))

File: scala-native/src/main/resources/scala-native/qt/quickcontrols2/QQuickStyle.cpp
#include <qt5/QtQuickControls2/QQuickStyle>
#include <qt5/QtCore/QString>
#include "QQuickStyle.hpp"

void q_quick_style_set_style(const q_string style) {
  QQuickStyle::setStyle(*((QString*) style));
  return;
}